ui/src/services/k8s/core.js

```diff
@@ -79,10 +78,12 @@ export async function queryPodInNamespace(namespace, podLabel) {
 
 export async function createNamespacedConfigMap(name, namespace, restProps) {
   const body = {
+    // it actually be removed by the k8s when the data is empty.
```

Comment:
Since it's removed, maybe you don't need to put it (anyway you have to create it with the first patch)